extensionsGallery.json

```diff
@@ -4828,6 +4828,75 @@
 					],
 					"statistics": [],
 					"flags": "preview"
+				},
+				{
+					"extensionId": "96",
+					"extensionName": "mark carrington.azuredatastudio-sql4cds",
+					"displayName": "SQL 4 CDS",
+					"shortDescription": "Query and modify data in Microsoft Dataverse/Dynamics 365/CRM using standard SQL queries.",
+					"publisher": {
+						"displayName": "Mark Carrington",
+						"publisherId": "mark carrington",
+						"publisherName": "Mark Carrington"
+					},
+					"versions": [
+						{
+							"version": "7.1.0",
+							"lastUpdated": "02/01/2023",
+							"assetUri": "",
+							"fallbackAssetUri": "fallbackAssetUri",
+							"files": [
+								{
+									"assetType": "Microsoft.VisualStudio.Services.VSIXPackage",
+									"source": "https://github.com/MarkMpn/Sql4Cds/releases/download/v7.1.0/azuredatastudio-sql4cds-7.1.0.vsix"
+								},
```

Comment:
Unfortunately, at this time we can only provide direct VSIX links to extensions we own. All 3rd party extensions need to use the `Microsoft.SQLOps.DownloadPage` asset type with a link to the download page only.


```suggestion
```

Reply:
OK, I've removed the vsix link and just left the URL to the GitHub release page for downloads.